.github/workflows/title_validation.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
```

Comment:
If the script contains errors, it should exit, as in fail early

Reply:
This way it skips the exit 123 part this way if the validation fails without further info.

Reply:
This way you know the returned error isn't intentional.

Reply:
That is the issue here, we want that script to fail for bad output, that is not a script error but an expected behavior.
Either the set -e  or the error handling in the end in unnecessary, because only one of them is executed.